cmd/disk-cache-backend.go

```diff
@@ -213,12 +217,46 @@ func (c *diskCache) diskAvailable(size int64) bool {
 	return int(usedPercent) < c.quotaPct
 }
 
+func (c *diskCache) toClear() uint64 {
```

Comment:
`// toClear returns how many bytes should be cleared to reach the low watermark quota.`
`// returns 0 if below quota.`
Actually not sure that is true ;)

Reply:
added tests for this function.